### DISCUSSION TITLE
Allow players to set up buying offers too, instead of selling offers only

### DISCUSSION THREAD

**Discussion by WeeIiam** [EpiCanard/GlobalMarketChest#119] [Ideas]
Hi there! Let me start off by thanking you for putting out such quality work. I love the plugin and it has been working wonders! I'd like to suggest a new feature. As of today, players are only able to put up **selling offers** with quantity and prices that the sellers determine. I'd love for buyers to be able through the same interface to put up **buying offers**: listing items that they would want to buy at a certain price. Is it something you could be considering to add in the future? Thanks for reading anyway. Keep up the good work and happy new year! 👾

**Comment by WeeIiam** [comment]
Hey! I came across your Trello page for GlobalMarketChest and I guess you've already listed the idea under _AskingShop/RequestShop_, so there I got my answer. Glad to know you're considering releasing this someday. Cheers :)

**Reply by EpiCanard** [reply]
Oh sorry I didn't see your message ^^ I never used the "Discussions" tab before :')

As you saw on Trello, the feature is already planned. I have a big refactor planned before, once done I hope I will be able to add more shop types more easier.
